### PR TITLE
Fixed more typos

### PR DESCRIPTION
**Pull request by bobmyhill**
Completed spell check in .cc and .h files

### CODE REVIEW

File: source/particle/integrator/interface.cc [hunk 1]
```diff
@@ -160,7 +160,7 @@ namespace aspect
                                "\n\n"
                                "As a consequence of these considerations, if you try to "
                                "assess convergence properties of an ODE integrator -- for "
-                               "example to verify that the RK4 integator converges with "
+                               "example to verify that the RK4 integrator converges with "
                                "forth order --, it is important to recall that the "
```
Comment (gassmoeller) [inline source/particle/integrator/interface.cc]: Please fix the forth as well.

File: source/postprocess/global_statistics.cc [hunk 2]
```diff
@@ -176,7 +176,7 @@ namespace aspect
             statistics.add_value("Number of nonlinear iterations",
                                  nonlinear_iterations);
 
-          // Only output statistics columns if the solver actually signaled at least one
+          // Only output statistics columns if the solver actually signalled at least one
```
Comment (gassmoeller) [inline source/postprocess/global_statistics.cc]: so you tried to sneak in British English after all :smile:

Comment (bobmyhill) [inline source/postprocess/global_statistics.cc]: Oh no!

File: source/particle/integrator/interface.cc (hunk 1 above)
Comment (bobmyhill) [inline source/particle/integrator/interface.cc]: Good catch!

File: source/postprocess/global_statistics.cc (hunk 2 above)
Comment (bangerth) [inline source/postprocess/global_statistics.cc]: I really had to look this one up because I would have spelled it with two L as well...

**Pull request merged**